features/step_definitions/mention_steps.rb

```diff
@@ -1,24 +1,24 @@
 And /^Alice has a post mentioning Bob$/ do
-  alice = User.find_by_email 'alice@alice.alice'
-  bob = User.find_by_email 'bob@bob.bob'
+  alice = User.find_by_email "alice@alice.alice"
+  bob = User.find_by_email "bob@bob.bob"
   aspect = alice.aspects.where(:name => "Besties").first
   alice.post(:status_message, :text => "@{Bob Jones; #{bob.person.diaspora_handle}}", :to => aspect)
 end
 
 And /^Alice has (\d+) posts mentioning Bob$/ do |n|
   n.to_i.times do
-    alice = User.find_by_email 'alice@alice.alice'
-    bob = User.find_by_email 'bob@bob.bob'
+    alice = User.find_by_email "alice@alice.alice"
+    bob = User.find_by_email "bob@bob.bob"
     aspect = alice.aspects.where(:name => "Besties").first
     alice.post(:status_message, :text => "@{Bob Jones; #{bob.person.diaspora_handle}}", :to => aspect)
   end
 end
 
 And /^I mention Alice in the publisher$/ do
-  alice = User.find_by_email 'alice@alice.alice'
-  write_in_publisher("@{Alice Smith ; #{alice.person.diaspora_handle}}")
+  write_in_publisher("@alice")
+  step %{I click on the first user in the mentions dropdown list}
```

Comment:
It's style, rubocop enforces a specific style. The other files were written before a specific style was enforced.